libs/application/templates/european-health-insurance-card/src/fields/ReviewScreen/ReviewScreen.tsx

```diff
@@ -0,0 +1,105 @@
+import { Address, NationalRegistry } from '../../lib/types'
+import { Box, Divider, Stack, Text } from '@island.is/island-ui/core'
+
+import { FC } from 'react'
+import { FieldBaseProps } from '@island.is/application/types'
+import { europeanHealthInsuranceCardApplicationMessages as e } from '../../lib/messages'
+import { formatText } from '@island.is/application/core'
+import { getFullName } from '../../lib/helpers/applicantHelper'
+import { useLocale } from '@island.is/localization'
+
+const ReviewScreen: FC<FieldBaseProps> = ({ application }) => {
+  const { formatMessage } = useLocale()
+
+  const nationalRegistryData = application.externalData.nationalRegistry
+    ?.data as NationalRegistry
+
+  const residence: Address = {
+    address: {
+      streetAddress: nationalRegistryData.address.streetAddress,
+      locality: nationalRegistryData.address.locality,
+      municipalityCode: nationalRegistryData.address.municipalityCode,
+      postalCode: nationalRegistryData.address.postalCode,
+    },
+  }
+
+  const plastic = application.answers.applyForPlastic as Array<any>
```

Comment:
We really want to avoid using type casting and especially with any. 
Id suggest implementing an interface for all the props used as it would make everything a lot cleaner.

Something like
```javascript
interface Plastic {
  // Define properties for plastic application objects
}

interface pdf {
  // Define properties for PDF application objects
}

interface ReviewScreenProps extends FieldBaseProps {
  application: {
    externalData: {
      nationalRegistry: {
        data: NationalRegistry
      }
    }
    answers: {
      applyForPlastic: PlasticApplication[]
      applyForPDF: PDFApplication[]
    }
  }
}
```